### PR TITLE
WIP Condense Lovelace configuration for easier reading

### PR DESCRIPTION
**Pull request by iantrich**
This is what I use in lots of my custom Lovelace cards and I find it much easier to read through than what we are currently doing and wanted some feedback on what we can do differently. Right now, the Lovelace configurations are a mess and nearly impossible to read with lots of redundancy even on the same page at times with things like the actions. Feedback/Ideas very much welcome.

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_lovelace/iframe.markdown [hunk 1]
```diff
@@ -15,25 +15,12 @@ You can't embed sites using HTTP if you are using HTTPS for your Home Assistant.
...
-  type: string
-  default: "50%"
-title:
-  required: false
-  description: The card title.
-  type: string
-{% endconfiguration %}
+| Name         | Required | Description        | Type   | Default |
+| ------------ | -------- | ------------------ | ------ | ------- |
+| type         | true     | `iframe`           | string | none    |
```
Comment (bramkragten) [inline source/_lovelace/iframe.markdown:20]:
```suggestion
| type         | true     | Card type          | string | `iframe`|
```

**Review by bramkragten — COMMENTED** [review]
Not sure, I do think our Lovelace documentation needs a massive update and it is too bulky, but a table might not be the best way forward as I think it is not so clear...

Especially our action config should be split out and linked to a general page, and maybe we can do that for more things.

**Comment by iantrich** [comment]
Any card that has entities or elements suffers greatly right now. It is really hard to read those, especially picture elements.

Maybe an initial list of possible elements that links to the docs is the clean/simple solution?

I agree with the action stuff needing to be split out for sure.

**Comment by frenck** [comment]
I don't think we should do this. This is working around an issue to make things more readable on a single place, instead of improving the configuration listing across all pages by changing the styling and/or plugin that processes it.

If we adjust this generally, all integrations and Lovelace configs benefit from that.

**Comment by iantrich** [comment]
Changing said styling is beyond my skillset :) But I"m glad that it is in your head now and hopefully you have some ideas. In the meantime, I'll likely start extracting redundant pieces to their own subpages for easier maintenance and cleaner pages

**Pull request closed without merging**